docs/advisories/etcd-manager-certificate-expiration.md

```diff
@@ -0,0 +1,64 @@
+# etcd-manager Certificate Expiration
+
+etcd-manager configures certificates for TLS communication between kube-apiserver and etcd, as well as between etcd members.
+These certificates are signed by the cluster CA and are valid for a duration of 1 year.
+
+Affected versions of etcd-manager did not handle the rotation of these certificates automatically.
+This means if the certificates expire, the etcd clusters and kube-apiserver will be inaccessible.
+This results in downtime of the cluster's control-plane.
+
+## How do I know if I'm affected?
+
+Clusters are affected by this issue if they're using a version of etcd-manager < `3.0.20200428`.
+The etcd-manager version is set automatically based on the Kops version.
+These Kops versions are affected:
+
+* Kops 1.10.0-alpha.1 through 1.16.1
+* Kops 1.17.0-alpha.1 through 1.17.0-beta.1
+* Kops 1.18.0-alpha.1 through 1.18.0-alpha.2
+
+The issue can be confirmed by checking for the existence of etcd-manager pods and observing their image tags:
+
+`kubectl get pods -n kube-system -l k8s-app=etcd-manager-main -o jsonpath='{..image}' | tr -s '[[:space:]]' '\n'`
+
+* If this outputs `kopeio/etcd-manager` images with tags older than `3.0.20200428`, the cluster is affected.
+* If this outputs an image other than `kopeio/etcd-manager`, the cluster may be affected.
+* If this does does not output anything or outputs `kopeio/etcd-manager` images with tags >= `3.0.20200428`, the cluster is not affected.
+
+## Solution
+
+etcd-manager versions >= `3.0.20200428` will request new certificates before they expire.
+
+Clusters can be upgraded to this version of etcd-manager in two ways. Both solutions require a rolling-update of the master nodes.
+
+1. Upgrade to Kops 1.16.2, 1.17.0-beta.2, or 1.18.1-alpha.3.
+   This is the recommended approach.
+   Follow the normal steps when upgrading Kops and confirm the etcd-manager image will be updated based on the output of `kops update cluster`.
+   ```
+   kops update cluster --yes
+   kops rolling-update cluster --instance-group-roles=Master
+   ```
+2. Another solution is to override the etcd-manager image in the ClusterSpec:
+   ```
+   kops edit cluster $CLUSTER_NAME
+   # Set `spec.etcdClusters[*].manager.image` to `kopeio/etcd-manager:3.0.20200428`
+   kops update cluster # confirm the image is being updated
+   kops update cluster --yes
+   kops rolling-update cluster --instance-group-roles=Master --force
+   ```
+
+## Workaround
```

Comment:
I would call this something like "Hack/workaround" or "Danger" because this doesn't solve anyone's problems, just saves them for tomorrow.